src/lib/components/backoffice/Sidebar/Sidebar.js

```diff
@@ -53,6 +54,19 @@ class Sidebar extends Component {
           <Overridable id="Sidebar.Menu">
             <>
               <Menu text vertical className="bo-menu">
+                <Menu.Item>
+                  <Menu.Header>Actions</Menu.Header>
+                  <Menu.Menu>
+                    <Menu.Item
+                      as={Link}
+                      active={checkinActive}
+                      to={BackOfficeRoutes.checkIn}
+                    >
+                      Check-in
```

Comment:
Could you add smaller font description under `Check-in` saying `Return copies` to provide more details what will happen under this menu item? A comment from a Library make us think that additional explanation might be useful for the librarians.